### PR TITLE
Update Server.schelp

### PR DESCRIPTION
**Pull request by tiagmoraismorgado**

### CODE REVIEW

**Comment by brianlheim** [comment]
@tiagmoraismorgado , it would be really helpful if you could lump these commits into the same branch and submit them as one pull request. You can also open a PR with one commit and continue committing to the same branch; the PR will be updated live with those additions. That way, it's easier to discuss related commits/issues, and the PR list doesn't get cluttered.

File: HelpSource/Classes/Server.schelp [hunk 1]
```diff
@@ -623,6 +623,13 @@ an optional instance of link::Classes/Condition:: used for evaluating this.
 argument:: ... args
 one or more valid OSC messages.
 
+method:: isLocal
+Returns true if current server is local
```
Comment (snappizz) [inline HelpSource/Classes/Server.schelp]: Doesn't add much. I suggest "Returns true if the server is local, i.e. running on the same machine as the language. `s.isLocal` is equivalent to `s.addr.isLocal`."

File: HelpSource/Classes/Server.schelp [hunk 2]
```diff
@@ -623,6 +623,13 @@ an optional instance of link::Classes/Condition:: used for evaluating this.
...
+code::
+~myLocalServer=Server.local;
+~myLocalServer.isLocal;
+::
+
```
Comment (snappizz) [inline HelpSource/Classes/Server.schelp]: This example isn't really necessary, we know how to call an instance method!

File: HelpSource/Classes/Server.schelp (hunk 1 above)
Comment (tiagmoraismorgado) [inline HelpSource/Classes/Server.schelp]: i removed the code example. an experienced supercollider user won't actually need much explanation in order to make something like this

Comment (telephon) [inline HelpSource/Classes/Server.schelp]: Could you please add an explanation, here is my suggestion:

Returns true if the server (scsynth) runs on the same computer as  the language (sclang).

**Comment by tiagmoraismorgado** [comment]
done

**Comment by vivid-synth** [comment]
This final diff looks good to me but can you submit it as a new PR without all the other commits? (Also with a clear commit message)

**Comment by gusano** [comment]
@vivid-synth You can also squash and merge if you want

File: HelpSource/Classes/Server.schelp [hunk 3]
```diff
@@ -623,6 +623,9 @@ an optional instance of link::Classes/Condition:: used for evaluating this.
 argument:: ... args
 one or more valid OSC messages.
 
+method:: isLocal
+Returns true if the server (scsynth) runs on the same computer as the language (sclang).
```
Comment (vivid-synth) [inline HelpSource/Classes/Server.schelp]: I'd also say "scsynth or supernova"

**Comment by brianlheim** [comment]
Closing, see #2645.

**Pull request closed without merging**